### ISSUE TITLE
Installation

### ISSUE THREAD

**Issue by llnnse** [rooi/homebridge-intercom-automation-hat#1]
So if I would want to edit which output is controlled, how would i do so? For example I would like to use the onboard relay to unlock/lock the door.
Should I just edit the python code and enter `sudo npm install` in the project directory?
Thanks

**Comment by rooi** [comment]
Yes, I've been wanting to make it configurable but I did not get a change to do this. Editing the Python code should work

**Comment by llnnse** [comment]
Another question, the bell I've assigned to my HomeKit home through your project shows a banner stating "Not Supported". Also I'm not able to choose to trigger automations with the bell. Do you somehow know why?

**Comment by rooi** [comment]
I have the same, in Apple's home app it says unsupported, but when I ring the bell it still receive a notification. I'm not sure if its a problem with homekit's intercom or the plugin, but I think the latter is more likely.

I did not got the change to investigate it further yet, but there is a workaround by using another homekit app (e.g. Home+) to setup the automation.

Now that at least you besides myself are using (or trying to) the plugin, this gives some motivation to fix and improve it. Perhaps you can help? On of the improvements I was think about was sending and receiving audio (I got one way working in a small test), but first things first

**Comment by llnnse** [comment]
I can help for sure, but I don't have any experience in programming homebridge plugins. Have you more details where the problem might occur from?
Getting to talk to the intercom when someone rings, was actually also an idea I had in my mind for some while. How did you manage to get audio passthrough? And in which direction?

**Comment by rooi** [comment]
Hi,

I would start with seeing how other doorbell/intercom plugins work and look like in apple's home app. I've seen some homebridge plugins from nello and ring.

If these show a supported icon in the homekit, there is probably something wrong in this plugin and the accessories that are created and we can compare the differences to hopefully fix it to gain a supported icon and, perhaps, other nice functionality that's missing e.g. a open button with the alerts in the ui

For the audio, I could send audio from the intercom to the door using a usb soundcard with mic and speaker connection (small device, I can look up the brand/type if you like). I think two way audio should work as well, but I didn't correct connected the wires. My guess is the ground for speaker and mic are not the single ground I expected and use for opening the door. Perhaps they are connected in series or something (I'm no electronics expert by the way ;)

**Comment by TheoPOZ** [comment]
Hi.  Anyone figured out how to change what pin triggers the relay or where to change it.  I had a look through the code but didn't see anything obvious to me.  Currently the door unlock is triggering BCM5 (sinking output 1 on the HAT), not BCM16 (relays on the HAT).

Help much appreciated.

**Comment by TheoPOZ** [comment]
Lo and behold I just figured it out.  Look in intercom-automation-hat.py and replace automationhat.output.one with  automationhat.relay.one or other output of choice.
